### PR TITLE
Reimplement with system serial port

### PR DESCRIPTION
**Pull request by afiebig**
This PR removes te dependency of the Wrapper and Dll Files. Now it uses System.Serial.Port

### CODE REVIEW

**Review by gonzunigad — APPROVED** [review]
we tested it and it works 🎉

File: README.md [hunk 1]
```diff
@@ -123,3 +123,48 @@ Linea de Comandos:
   ```bash
   dotnet.exe nuget push TransbankPosSDK/bin/Release/TransbankPosSDK.<version>.nupkg -k <APIKEY> -s https://api.nuget.org/v3/index.json
   ```
+
+## Contribuciones ✨
+
+Agradecimientos especiales a quienes nos ayudan a mejorar esta libreria.
```
Comment (gonzunigad) [inline README.md:129]:
```suggestion
Agradecimientos especiales a quienes nos ayudan a mejorar esta librería.
```

**Review by gonzunigad — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -123,3 +123,48 @@ Linea de Comandos:
...
+
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/DarkFrostnight"><img src="https://avatars.githubusercontent.com/u/36648048?v=4" width="100px;" alt=""/><br /><sub><b>Mauricio Astudillo</b></sub></a><br /><a href="#bugs-DarkFrostnight" title="Reportar Bugs">🐛</a> <a href="userTest-DarkFrostnight" title="Pruebas de Usuario">📓</a> <a href="ideas-DarkFrostnight" title="Nuevas Ideas">🤔</a>
+    </td>
+  </tr>
+</table>
+
+### Simbologia
```
Comment (gonzunigad) [inline README.md:138]:
```suggestion
### Simbología
```

File: README.md [hunk 3]
```diff
@@ -123,3 +123,48 @@ Linea de Comandos:
...
+<table>
+  <tr>
+    <td align="center">
+      💻 <br> Codigo
```
Comment (gonzunigad) [inline README.md:142]:
```suggestion
      💻 <br> Código
```

File: TransbankPosSDK/Responses/DetailResponse.cs [hunk 4]
```diff
@@ -27,69 +18,167 @@ public class DetailResponse
...
+                   "Shares Amount: " + SharesAmount + "\n" +
+                   "Last 4 Digits: " + Last4Digits + "\n" +
+                   "Operation Number: " + OperationNumber + "\n" +
+                   "Card Type: " + CardType + "\n" +
+                   "Accounting Date: " + formatedAccountingDate + "\n" +
+                   "Account Number: " + AccountNumber + "\n" +
+                   "Card Brand: " + CardBrand + "\n" +
+                   "Real Date: " + formatedRealDate + "\n" +
+                   "Employee Id: " + EmployeeId + "\n" +
+                   "Tip: " + Tip;
```
Comment (gonzunigad) [inline TransbankPosSDK/Responses/DetailResponse.cs:180]:
```suggestion
            string formattedAccountingDate = AccountingDate.HasValue ? AccountingDate.Value.ToString("dd/MM/yyyy hh:mm:ss") : "";
            string formattedRealDate = RealDate.HasValue ? RealDate.Value.ToString("dd/MM/yyyy hh:mm:ss") : "";
            return "Function: " + FunctionCode + "\n" +
                   "Response: " + ResponseMessage + "\n" +
                   "Commerce Code: " + CommerceCode + "\n" +
                   "Terminal Id: " + TerminalId + "\n" +
                   "Ticket: " + Ticket + "\n" +
                   "AuthorizationCode Code: " + AuthorizationCode + "\n" +
                   "Amount: " + Amount + "\n" +
                   "Shares Number: " + SharesNumber + "\n" +
                   "Shares Amount: " + SharesAmount + "\n" +
                   "Last 4 Digits: " + Last4Digits + "\n" +
                   "Operation Number: " + OperationNumber + "\n" +
                   "Card Type: " + CardType + "\n" +
                   "Accounting Date: " + formattedAccountingDate + "\n" +
                   "Account Number: " + AccountNumber + "\n" +
                   "Card Brand: " + CardBrand + "\n" +
                   "Real Date: " + formattedRealDate + "\n" +
                   "Employee Id: " + EmployeeId + "\n" +
                   "Tip: " + Tip;
```

**Review by gonzunigad — CHANGES_REQUESTED** [review]

File: TransbankPosSDK/Utils/ResponseCodes.cs [hunk 5]
```diff
@@ -8,53 +8,55 @@ public static class ResponseCodes
...
-            {07, "Transacción Cancelada desde el POS"},
-            {08, "No puede Anular Transacción Debito"},
-            {09, "Error Lectura Tarjeta"},
+            {02, "Host no responde"},
+            {03, "Conexión falló"},
+            {04, "Transacción ya fue anulada"},
+            {05, "No existe transacción para anular"},
+            {06, "Tarjeta no soportada"},
+            {07, "Transacción cancelada desde el POS"},
+            {08, "No puede anular transacción debito"},
```
Comment (gonzunigad) [inline TransbankPosSDK/Utils/ResponseCodes.cs:17]:
```suggestion
            {08, "No puede anular transacción débito"},
```

File: TransbankPosSDK/Utils/ResponseCodes.cs [hunk 6]
```diff
@@ -8,53 +8,55 @@ public static class ResponseCodes
...
-            {16, "Error Formato Respuesta del HOST"},
+            {14, "No hay tono"},
+            {15, "Archivo BITMAP.DAT no encontrado. favor cargue"},
+            {16, "Error formato respuesta del host"},
             {17, "Error en los 4 últimos dígitos."},
             {18, "Menú invalido"},
             {19, "ERROR_TARJ_DIST"},
-            {20, "Tarjeta Invalida"},
-            {21, "Anulación. No Permitida"},
+            {20, "Tarjeta invalida"},
```
Comment (gonzunigad) [inline TransbankPosSDK/Utils/ResponseCodes.cs:29]:
```suggestion
            {20, "Tarjeta inválida"},
```

File: TransbankPosSDK/Utils/ResponseCodes.cs [hunk 7]
```diff
@@ -8,53 +8,55 @@ public static class ResponseCodes
...
-            {61, "Error en Armado de Solicitud"},
-            {62, "Problema con el Pinpad interno"},
-            {65, "Error al Procesar la Respuesta del Host"},
-            {67, "Superó Número Máximo de Ventas, Debe Ejecutar Cierre"},
-            {68, "Error Genérico, Falla al Ingresar Montos"},
-            {70, "Error de formato Campo de Boleta MAX 6"},
-            {71, "Error de Largo Campo de Impresión"},
-            {72, "Error de Monto Venta, Debe ser Mayor que 0"},
+            {24, "Impresora sin papel"},
+            {25, "Fecha invalida"},
```
Comment (gonzunigad) [inline TransbankPosSDK/Utils/ResponseCodes.cs:33]:
```suggestion
            {25, "Fecha inválida"},
```

**Review by gonzunigad — CHANGES_REQUESTED** [review]

**Review by gonzunigad — DISMISSED** [review]
🎉

File: README.md (hunk 1 above)
Comment (afiebig) [inline README.md]:
```suggestion
Agradecimientos especiales a quienes nos ayudan a mejorar esta librería.
```

**Review by gonzunigad — APPROVED** [review]
🎉

**Pull request merged**